lib/Domain/Fixer/DummyFixer.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Phpactor\CodeBuilder\Domain\Fixer;
+
+use Phpactor\CodeBuilder\Domain\StyleFixer;
+
+class DummyFixer implements StyleFixer
```

Comment:
```suggestion
class NullFixer implements StyleFixer
```

Personally `DummyFixer` makes me think of a test double, I think `NullFixer` might be more accurate since it respects the general convention around the [null object pattern](https://en.wikipedia.org/wiki/Null_object_pattern)